CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## 1.0.2 (TBD)
+
+### Features
+- Amazon DocumentDB module
+- Amazon Neptune module
+- Amazon Neptune Slow query module
+- Amazon RDS MariaDB module
+- Amazon RDS MySQL module
+- Amazon RDS MySQL Slow query module
+- Amazon RDS Oracle (standard auditing) module
+- Amazon RDS Oracle (unified auditing) module
+- Amazon Redshift module
```

Comment:
Do we want to specify both ODBC and S3 versions for Redshift? if we're already differentiating between regular and slow query modules and the different auditing types for RDS Oracle